packages/desktop-client/src/components/modals/ConfirmTransactionDelete.tsx

```diff
@@ -48,16 +49,18 @@ export function ConfirmTransactionDelete({
               >
                 Cancel
               </Button>
-              <Button
-                variant="primary"
-                style={narrowButtonStyle}
-                onPress={() => {
-                  onConfirm();
-                  close();
-                }}
-              >
-                Delete
-              </Button>
+              <InitialFocus>
```

Comment:
@psybers - I'm not questioning you on this, just a learning moment for me as I get to better grips with React. Why  is this one wrapped in <InitialFocus> instead of an autofocus prop?

Reply:
This uses the `focus()` call on the element, after a short delay.  Which is needed in some cases.